video/out/w32_common.c

```diff
@@ -1514,8 +1532,19 @@ static void gui_thread_reconfig(void *ptr)
     }
 
     // Save new window size and position.
-    const int x = rc->left + rect_w(*rc) / 2 - vo->dwidth / 2;
-    const int y = rc->top + rect_h(*rc) / 2 - vo->dheight / 2;
+    int x, y;
+    if (w32->snapped_left)
+        x = rc->left;
+    else if (w32->snapped_right)
+        x = rc->right - vo->dwidth;
+    else
+        x = rc->left + rect_w(*rc) / 2 - vo->dwidth / 2;
+    if (w32->snapped_top)
+        y = rc->top;
+    else if (w32->snapped_bottom)
+        y = rc->bottom - vo->dheight;
+    else
+        y = rc->top + rect_h(*rc) / 2 - vo->dheight / 2;
```

Comment:
Please add some parentheses or empty lines for readability.

Reply:
Done. Wrapped it in a function.